src/indigenousLayers.js

```diff
@@ -1,13 +1,12 @@
-L.LayerGroup.IndigenousLandsTerritoriesLayer = L.LayerGroup.extend(
+L.LayerGroup.IndigenousLayers = L.LayerGroup.extend(
 
     {
+
         options: {
-            url: 'https://native-land.ca/api/index.php?maps=territories&position=45,-72',
+            url: this.url,
```

Comment:
Let's replace this `name: this.name ,`

Reply:
I'll remove the url being passed, and pass only the name of the layer as parameter.
I'll replace this section with an if else block like done in the other sections.